MinecraftClient/Mapping/Entity.cs

```diff
@@ -54,6 +54,18 @@ public class Entity
         /// </summary>
         public Location Location;
 
+        /// <summary>
+        /// Entity head yaw
+        /// </summary>
+        /// <remarks>Untested</remarks>
```

Comment:
Is it still untested? 😜

Reply:
Yes. I am too tired to test it right now.